frontend/helpers/cache-control-helpers.ts

```diff
@@ -3,11 +3,21 @@ import { GetServerSidePropsMiddleware } from '@lib/next-middleware';
 import { Duration } from '../lib/duration';
 import { GetServerSidePropsContext } from 'next';
 
-interface CacheControlOptions {
+interface EnabledCacheControlOptions {
    sMaxAge: number;
    staleWhileRevalidate: number;
 }
 
+interface DisabledCacheControlOptions {
```

Comment:
Maybe this should be a const instead of a type? Not entirely sure that'd work out, but since it's prescriptive in its key and value, seems like a const.

Reply:
Yea, I took a look at using `false` or an object like `const obj = {disabled: true}` and it felt worse to use with the type system then this odd type. I am inclined to leave it and we can take a stab at changing it later.